css_tasks/task10/index.html

```diff
@@ -261,25 +261,35 @@ <h2 class="header__title header__title--center header__title--form">
           Over 1000 designers are using ...
         </h2>
       </header>
-      <form action="" class="container inner-container ">
-        <input
-          class="inner-container__element inner-container__element--form"
-          type="text"
-          placeholder="Full Name"
-          name="name"
-        />
-        <input
-          class="inner-container__element inner-container__element--form"
-          type="email"
-          placeholder="email"
-          name="email"
-        />
-        <input
-          class="inner-container__element inner-container__element--form"
-          type="password"
-          placeholder="password"
-          name="password"
-        />
+      <form
+        action=""
```

Comment:
at least post to not see all typed stuff at url, all some good ux with cleaning form fields and showing some message to user that form was sended

Reply:
Changed